scripts/contentScript.js

```diff
@@ -639,7 +741,7 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 						"repo_owner": ownerName,
 						"repo_name": repoName,
 						"user_id": userId,
-						"is_github": false
+						"repo_provider": 'bitucket'
```

Comment:
```suggestion
						"repo_provider": 'bitbucket'
```
Please mind the spelling. 
A better approach would be use fixed constants at the top of the file:

```
const Provider = Object.freeze({
    GITHUB: 'github',
    BITBUCKET: 'bitbucket',
    GITLAB: 'gitlab'
}
```

and then use that wherever you are using a `repo_provider`:

`repo_provider = Provider.GITHUB`